.github/workflows/ci.yml

```diff
@@ -65,6 +65,9 @@ jobs:
           - monorepo with npm
           - monorepo with yarn
           - monorepo with pnpm
+
+          # build-only tests for testing if the rollup config works at all
+          - rollup-build
```

Comment:
new style of tests -- asserting against fixtures -- allows us to be more specific, while still keeping the full e2e tests in tact for covering ... e2e